Automattic/woocommerce-payments
# Add wordpress/data hooks to the general-settings component

Fixes #1727

#### Changes proposed in this Pull Request

Replaces the internal state management in `general-settings` component and uses instead `@wordpress/data` hooks. The updated hook allows to display and update the fields `manual_capture` and `account_statement_descriptor`.

#### Testing instructions

1. Ensure you have enabled the flag for grouped settings.
2. Go to WCPay settings [here](http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments).
3. Enable the checkbox `Issue an authorization on checkout, and capture later`.
4. Set a custom bank statement in the input `Customer bank statement`.
5. Press the `Save changes` button.
6. Reload the screen.
7. The changes should have persisted after reloading the page.

-------------------
